### PR TITLE
Highlight prerequisites line on amdgpu-install page

### PR DESCRIPTION
**Pull request by alexxu-amd**
Multiple feedbacks from internal and external mention the "set user group permission" command is not directly available from amdgpu-install page, which in fact, is included in the prerequisites page but tends to get ignored. Highlight the prerequisites line for improved visibility.

### CODE REVIEW

File: docs/install/amdgpu-install.rst [hunk 1]
```diff
@@ -22,7 +22,9 @@ Installation
 
 Installation of ``amdgpu-install`` differs slightly depending on the OS and its package manager.
 
-Make sure that the :doc:`/install/prerequisites` are met before installing.
+.. note::
```
Comment (yhuiYH) [inline docs/install/amdgpu-install.rst]:
```suggestion
.. important::
```

Do you want to try changing from "note" to "important"?  Do we want it to standout more?

**Comment by yhuiYH** [comment]
Also, to be complete, we should add a similar note on the "Ubuntu, SLES, and RHEL" install pages.  Can you add that to your PR?

File: docs/install/amdgpu-install.rst (hunk 1 above)
Comment (alexxu-amd) [reply docs/install/amdgpu-install.rst]: Yea, I agree. Wasn't aware of the important block

**Comment by alexxu-amd** [comment]
Added the notes for Ubuntu, SLES, and RHEL. Also added some minor changes to unify the content:
- RHEL refactors the installation part to install-rocm-template.rst, whereas the other two don't
- RHEL uses regular tense on titles, the other two uses present tense
- SLSE refers repo as "ROCm repositories", the other two just uses "repositories"

File: docs/install/native-install/ubuntu.rst [hunk 2]
```diff
@@ -86,6 +86,10 @@ Add the ROCm repository.
 Installing
 ================================================
 
+.. important::
```
Comment (yhuiYH) [inline docs/install/native-install/ubuntu.rst]: @dbieleck Do you like where @alexxu-amd added it in the "Installing" part?  Or do you think it should be right at the top of the page?

I think I'd like it right up at line 8, as the first thing they see, even before they start "registering the repositories".

Comment (dbieleck) [reply docs/install/native-install/ubuntu.rst]: I agree.  It might be better up at line 8 so it's visible right away.

**Review by yhuiYH — APPROVED** [review]
OK, this LGTM now.  Approved.

**Pull request merged**